num1 = 20
num2 = 0

"""Estructura del try catch en python"""
try:
    tem = num1/num2
except:
    print("Error en el programa. Revisa con el administrador de sistemas")
finally:
    print("Siempre me voy a aparecer en la noche")